### PR TITLE
Feature/eth test coverage plt 183

### PR DESCRIPTION
**Pull request by jakcinmarina**

### CODE REVIEW

**Comment by pfranc** [comment]
Task linked: [PLT-183 [SDK] Cover with tests ETH](https://app.clickup.com/t/24443045/PLT-183)

File: packages/shared/testing/evm-based/src/lib/evm-based.erc721.test-factory.ts [hunk 1]
```diff
@@ -427,6 +508,84 @@ export const erc721TestFactory = {
...
+              fee: {
+                gasLimit: '3264521',
+                gasPrice: '20',
+              },
+            },
+            provider,
+          )
+          fail()
+        } catch (e: any) {
+          expect(e.reason).toMatch('invalid address')
```
Comment (npwork) [inline packages/shared/testing/evm-based/src/lib/evm-based.erc721.test-factory.ts:584]: Take a look at 
packages/shared/testing/btc-based/src/lib/btc-based.tx.test-factory.ts

).rejects.toThrowSdkErrorWithCode(SdkErrorCode.BTC_FEE_TOO_SMALL)

File: packages/shared/testing/evm-based/src/lib/evm-based.erc721.test-factory.ts [hunk 2]
```diff
@@ -349,6 +349,87 @@ export const erc721TestFactory = {
...
+              authorAddresses: [address],
+              fee: {
+                gasLimit: '3264521',
+                gasPrice: '20',
+              },
+            },
+            provider,
+          )
+          fail()
+        } catch (e: any) {
```
Comment (npwork) [inline packages/shared/testing/evm-based/src/lib/evm-based.erc721.test-factory.ts:427]: same here

**Review by npwork — APPROVED** [review]

File: packages/shared/testing/evm-based/src/lib/evm-based.erc721.test-factory.ts (hunk 1 above)
Comment (jakcinmarina) [reply packages/shared/testing/evm-based/src/lib/evm-based.erc721.test-factory.ts:584]: I tried implementing something similar but it was throwing me this error while testing
image
The code in the error is INVALID_ARGUMENT but it's not working...

**Pull request merged**